plugins/woocommerce/phpstan.neon

```diff
@@ -0,0 +1,27 @@
+includes:
+	- phpstan-baseline.neon
+
+parameters:
+	tmpDir: tmp
+	level: 9
+	phpVersion: 80400
+	paths:
+		- woocommerce.php
+		- src/
+		- includes/
+	excludePaths:
+		# Matches the prior test implementation; GeoIP relies on data files.
+		- includes/class-wc-geo-ip.php
+		- includes/react-admin/feature-config.php (?)
```

Comment:
Exclude this as it is `gitignored`, also added the `(?)` as it can exist locally.